platforms/intellij/compose/playground/build.gradle.kts

```diff
@@ -56,3 +66,7 @@ configurations
   .configureEach { attributes { attribute(KotlinPlatformType.attribute, KotlinPlatformType.jvm) } }
 
 dependencies { lintChecks(libs.composeLints) }
+
+tasks.withType<KotlinCompile>().configureEach {
+  compilerOptions { freeCompilerArgs.add("-Xcontext-receivers") }
```

Comment:
Let's add a comment explaining why this is needed, and let's limit it to test JVM compilations.

```kotlin
tasks.named { it.name == "compileJvmTestKotlin" }...
```

Should be soooomething like that

Reply:
Do I need to register `compileJvmTestKotlin` task? It's not in the list of gradlew tasks.

Reply:
the name might be slightly different, you'll want to find the task name for the task that compiles test jvm sources

Reply:
I believe it could be `jvmTestClasses` under the `./gradlew tasks`

`jvmTestClasses - Assembles outputs for compilation 'test' of target 'jvm'` and added the change here: fff72b9fa44861e4a35431a293989dfacb0903dc

Reply:
jvmTestClasses is not a type of `KotlinCompile` as far as I know. Do you see this actually being configured there? I looked and I'm think what you want is `compileTestKotlinJvm`, but if it works without adding this opt-in at all then we can omit it?

Reply:
I found the task `compileTestKotlinJvm` while running `./gradlew tasks --all `, so I committed it here: e1ebfee38af50a0dc12f7af6b590b77d48504466. Is this the task you are looking for?

Reply:
yes, in general you are gonna be better off not relying on `./gradlew tasks` vs just running a task you know will run it and adding `--dry-run` to the end, which will just print all the tasks it's gonna run. You can also look at a build scan to see task types.